draft-ietf-httpbis-cache-latest.xml

```diff
@@ -681,7 +681,7 @@
 </t>
 <ul>
    <li>Although all date formats are specified to be case-sensitive, 
```

Comment:
AFAIU, we now list all parts of the ABNF that are not digits or delimiters. So why not simply match the whole string case-insensitively?

Reply:
Fair point.